testing/sdk_recovery.py

```diff
@@ -43,7 +48,8 @@ def check_permanent_recovery(
     sdk_plan.wait_for_kicked_off_recovery(service_name, recovery_timeout_s)
     sdk_plan.wait_for_completed_recovery(service_name, recovery_timeout_s)
 
-    sdk_tasks.check_tasks_updated(service_name, pod_name, tasks_to_replace)
+    for pod, tasks in tasks_to_replace.items():
```

Comment:
s/items/iteritems/?

Reply:
from: https://stackoverflow.com/questions/10458437/what-is-the-difference-between-dict-items-and-dict-iteritems
> One of Python 3’s changes is that  items() now return iterators, and a list is never fully built. The iteritems() method is also gone, since items() in Python 3 works like viewitems() in Python 2.7.

Also:
```python
✗ ipython
Python 3.6.4 (default, Mar  9 2018, 23:15:03)
Type 'copyright', 'credits' or 'license' for more information
IPython 6.1.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: foo = {'a': 1, 'b': 2}

In [2]: for f, v in foo.iteritems():
   ...:     print(f, v)
   ...:
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-2c0163c99ec9> in <module>()
----> 1 for f, v in foo.iteritems():
      2     print(f, v)
      3
```